source/common/config/unified_mux/subscription_state.h

```diff
@@ -0,0 +1,132 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "envoy/common/pure.h"
+#include "envoy/config/subscription.h"
+#include "envoy/event/dispatcher.h"
+#include "envoy/service/discovery/v3/discovery.pb.h"
+
+#include "source/common/config/ttl.h"
+#include "source/common/config/update_ack.h"
+#include "source/common/config/utility.h"
+#include "source/common/protobuf/protobuf.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Config {
+namespace UnifiedMux {
```

Comment:
As a total nit, I think `UnifiedMux` makes sense to us right now, but in the future is a less useful name. Maybe just something descriptive like `XdsMux` (assuming we will kill the GrpcMux and NewGrpcMux over time).

Reply:
renamed.